Adding Scramble Secrecy Items to the Regs (#45)
CarterKoala/wca-regulations-wrc: wca-regulations.md

- 1a) A competition must include a WCA Delegate and an organization team (consisting of one or more individuals) with the following officials: judges, scramblers and score takers.
- 1b) The organization team of a competition is responsible for logistics before, during, and after the competition.
- 1c) The WCA Delegate is responsible for ensuring that the competition adheres to the WCA Regulations and any applicable WCA policies or requirements. The WCA Delegate may appoint other members of the organization team to carry out specific responsibilities on their behalf, but is ultimately accountable for how these responsibilities are carried out.

>     - 1c1) The WCA Delegate must ensure that the competition follows the [WCA Scramble Accountability Policy](https://documents.worldcubeassociation.org/documents/policies/external/Scramble%20Accountability.pdf).

Reviewer: Why are we removing this? It's mostly a reminder for delegates.

Author: Because there isn't much reason to say this and not add e.g. competition requirements policy or anything else - that is already covered in 1c

        - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise (e.g. the Stackmat timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.
    - A1c) A competitor participating in an event must be able to fulfill the event's requirements (see [Regulation 2k6](regulations:regulation:2k6)).
- A2) Scrambling:

>     - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.
>         - A2a1) If a Competitor Waiting Area is in use the competitor must remain within within their assigned Competitor Waiting Area, except when called to perform an attempt, until they have completed all their attempts in the round, unless permission to leave has been granted by the WCA Delegate. Penalty: disqualification of any remaining attempts (DNF) at the discretion of the WCA Delegate. 

Reviewer: ```suggestion
    - A2a) When called for a round, the competitor submits a solved puzzle to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.
        - A2a1) If a Competitor Waiting Area is in use, the competitor must remain within their assigned Competitor Waiting Area until they are called to complete an attempt. The competitor must return to their assigned Competitor Waiting Area after completing each attempt, until they have completed all their attempts in the round. Exception: competitors may leave the Competitor Waiting Area with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate. 
```

>         - A2a1) If a Competitor Waiting Area is in use, the competitor must remain within their assigned Competitor Waiting Area until they are called to complete an attempt. The competitor must return to their assigned Competitor Waiting Area after completing each attempt, until they have completed all their attempts in the round. Exception: competitors may leave the Competitor Waiting Area with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate. 

Author: ```suggestion
        - A2a1) If a Competitor Waiting Area is in use, the competitor must remain within their assigned Competitor Waiting Area until they are called to complete an attempt. The competitor must return to their assigned Competitor Waiting Area after completing each attempt if they have more attempts in the round. Exception: competitors may leave the Competitor Waiting Area with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate. 
```

>         - A2a2) If a Competitor Waiting Area is not in use, the competitor must remain at their assigned solving station until they have completed all their attempts in the round, unless permission to leave has been granted by the WCA Delegate. Penalty: disqualification of any remaining attempts (DNF) at the discretion of the WCA Delegate.

Reviewer: ```suggestion
        - A2a2) If a Competitor Waiting Area is not in use, the competitor must remain at their assigned solving station until they have completed all their attempts in the round. Exception: competitors may leave their assigned solving station with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate.
```

        - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise (e.g. the Stackmat timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.
    - A1c) A competitor participating in an event must be able to fulfill the event's requirements (see [Regulation 2k6](regulations:regulation:2k6)).
- A2) Scrambling:
    - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.

>         - A2a1) If a Competitor Waiting Area is in use the competitor must remain within within their assigned Competitor Waiting Area, except when called to perform an attempt, until they have completed all their attempts in the round, unless permission to leave has been granted by the WCA Delegate. Penalty: disqualification of any remaining attempts (DNF) at the discretion of the WCA Delegate. 

Author: So many commas in this sentence. I'm not sure right now how to make it better though

>         - A2a1) If a Competitor Waiting Area is not in use, the competitor must remain at their assigned solving station until they have completed all their attempts in the round, unless permission to leave has been granted by the WCA Delegate. Penalty: disqualification of any remaining attempts (DNF) at the discretion of the WCA Delegate.

Author: ```suggestion
        - A2a2) If a Competitor Waiting Area is not in use, the competitor must remain at their assigned solving station until they have completed all their attempts in the round, unless permission to leave has been granted by the WCA Delegate. Penalty: disqualification of any remaining attempts (DNF) at the discretion of the WCA Delegate.
```

        - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise (e.g. the Stackmat timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.
    - A1c) A competitor participating in an event must be able to fulfill the event's requirements (see [Regulation 2k6](regulations:regulation:2k6)).
- A2) Scrambling:

>     - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. If a Competitor Waiting Area is in use the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitors Waiting Area in not in use the competitor then waits at their assigned solving station.

Author: ```suggestion
    - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.
```
